src/tests/ftest/erasurecode/offline_rebuild_single.yaml

```diff
@@ -10,6 +10,9 @@ hosts:
 setup:
   start_servers_once: False
 timeout: 900
+agent_config:
+  #cache_expiration: 1
+  disable_caching: true
```

Comment:
Is this an optimization or necessary for these tests? In other words, what would happen with a similar workflow in production?

Reply:
I made these .yaml file changes to address intermittent overall test timeout failures in some in Features: rebuild weekly regression tests. As seen by libdaos programs (like the 'daos' command) getting cart proto query RPC timeouts in the libdaos bootstrapping code. When it randomly chooses a stopped engine as the destination of that RPC. But I really wanted those tests to be run with my engine C code changes, to make sure they did not accidentally introduce a regression.

It is a similar change to what was done for other tests https://github.com/daos-stack/daos/pull/14086
A longer term solution would be to improve the bootstrapping code logic in libdaos to shorten timeouts and try alternate engines. Some discussion in https://daosio.atlassian.net/browse/DAOS-15440 and several other tickets and discussion channels IIRC.

Reply:
Gotcha. Since we're tracking an improvement, no issue from me

Reply:
I thought @mjmac was using this to workaround an issue where slurm replaces the user/group stuff.  You may want to sync with him.